### PR TITLE
Feature/template integration

### PR DESCRIPTION
**Pull request by arqamtahir**

### CODE REVIEW

**Comment by MShahnoor** [comment]
Farigh he

**Comment by hamza5999** [comment]
better luck next time

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -13,3 +13,4 @@
  *= require_tree .
  *= require_self
```
Comment (moizafzal936) [inline app/assets/stylesheets/application.scss:14]: Change commit name

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -12,3 +12,6 @@ Rails.start()
 Turbolinks.start()
 ActiveStorage.start()
 
+
+window.$ = window.jQuery = require("jquery");
```
Comment (moizafzal936) [inline app/javascript/packs/application.js:16]: Add different commit for jquery.

File: config/webpacker.yml [hunk 3]
```diff
@@ -10,7 +10,7 @@ default: &default
 
   # Additional paths webpack should lookup modules
   # ['app/assets', 'engine/foo/app/assets']
-  additional_paths: []
+  additional_paths: ["vendor/assets/javascripts/dashboard"]
```
Comment (moizafzal936) [inline config/webpacker.yml:13]: Change commit name

File: app/javascript/packs/application.js [hunk 4]
```diff
@@ -12,3 +12,6 @@ Rails.start()
 Turbolinks.start()
 ActiveStorage.start()
 
+
```
Comment (moizafzal936) [inline app/javascript/packs/application.js:15]: Change the commit name

File: app/assets/stylesheets/home.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the home controller here.
```
Comment (moizafzal936) [inline app/assets/stylesheets/home.scss]: Additional file!

File: app/helpers/home_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```
Comment (moizafzal936) [inline app/helpers/home_helper.rb:1]: Additional File!

File: test/controllers/home_controller_test.rb [hunk 7]
```diff
@@ -0,0 +1,8 @@
+require "test_helper"
```
Comment (moizafzal936) [inline test/controllers/home_controller_test.rb:1]: Additional File

File: config/routes.rb [hunk 8]
```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
```
Comment (moizafzal936) [inline config/routes.rb:1]: Not relevant to commit.

File: app/views/dashboard/_content-section.html.erb [hunk 9]
```diff
@@ -0,0 +1,6 @@
+<section class="page-content container-fluid">
+                        <%= render 'dashboard/stats/card' %>
+                        <%= render 'dashboard/stats/chart' %>
```
Comment (moizafzal936) [inline app/views/dashboard/_content-section.html.erb:3]: Indentation need to be fixed

**Review by moizafzal936 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**